### DISCUSSION TITLE
Minimum and Maximum Target Temperature

### DISCUSSION THREAD

**Discussion by bjappler** [revk/xESP32-Faikout#412] [Ideas]
Would it be possible to set a minimum and maximum target temperature instead of using 0,5 1 and 2 C in faikin automatic modus?
The automatic mode should work as follows:

If the temperature falls below the minimum temperature, heat up to the maximum temperature. Only heat again when the temperature falls below the minimum temperature. 
This would have the advantage that the air conditioning system would have a low cycle rate. 

Is this setting already possible or could be a feature update?

It would also be great if the Faikin auto mode only referred to the temperature, but other parameters such as the power of the outdoor unit or ventilation level could be set manually. This would also reduce the cycle rate. 

Many thanks in advance.

**Comment by revk** [comment]
The manual covers the way it works in detail. It has various modes you can set, which cover ways of working like that.

**Reply by bjappler** [reply]
Would these settings be possible in your faikin http front end? I did not find this in your manuals.

**Reply by revk** [reply]
All the settings are on the web pages.

There is not a specific min/max but the e 0.5, 1, 2 are a min/max that far each side of the temp you set. Do you need wider than 4C?

The extra settings control how it uses these, and there is definitely stuff in the manual on that.

**Comment by bjappler** [comment]
I found the variable pushtemp in the instructions. I assume this is what I am looking for, if I have understood it correctly. 

Would a value of 2.0 mean that with a target temperature of 20 degrees and Faikin auto mode -+ 2 degrees, the air conditioning heats up to 20 degrees and only heats again when the temperature falls below 18 degrees?

This would be the feature I was looking for!

**Reply by revk** [reply]
20 +/- 2 means a min 18 and a max 22.

There is a thermostat setting as well. The manual does cover this in some detail, honest.

**Comment by bjappler** [comment]
This is what I was looking for! Thank you so much for your help and for your patience.
